apps/dsp-app/src/assets/i18n/en.json

```
@@ -160,6 +160,7 @@
     "versionNotValid": "The version you requested is not valid. The latest version is displayed.",
     "notFound": "This resource is not found.",
     "deleted": "This resource has been deleted.",
+    "unauthorized": "You do not have access to this resource.",
```

```suggestion
    "unauthorized": "You do not have the access to this resource.",
```
You can also apply this little change to other locale files, but don't have too (as they going to be translated anyway).